src/components/SideMenu/SideMenu.test.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  fireEvent,
+  getByTestId,
+  getByText,
+  render,
+  screen,
+} from "@testing-library/react";
+import AdminHomePage from "../../pages/AdminHomePage/AdminHomePage";
+import { MemoryRouter } from "react-router";
+import { routes } from "../../constants/routes";
+
+describe("Test side-menu", () => {
+  test("should render side menu on admin page", async () => {
+    render(
+      <MemoryRouter
+        initialEntries={[
+          {
+            pathname: routes.HOME,
+          },
+        ]}
+      >
+        <AdminHomePage />
+      </MemoryRouter>
+    );
+
+    const menuComponent = await screen.findByTestId(`menu-list`);
+
+    expect(menuComponent).toBeTruthy();
+  });
+
+  test("should change selected item on click", () => {
+    const { getByTestId } = render(
+      <MemoryRouter
+        initialEntries={[
+          {
+            pathname: routes.HOME,
+          },
+        ]}
+      >
+        <AdminHomePage />
+      </MemoryRouter>
+    );
+
+    const readerButton = getByTestId("reader-button");
```

Comment:
Use screen.getByTestId function instead of getting it from render, same way you did in the test above this one